website/content/docs/connect/ca/vault.mdx

```diff
@@ -181,174 +163,171 @@ The configuration options are listed below.
+  }
 
-path "/sys/mounts/<root_pki_path>" {
-  capabilities = [ "read" ]
-}
+  path "/sys/mounts/<intermediate_pki_path>" {
+    capabilities = [ "read" ]
+  }
 
-path "/sys/mounts/<intermediate_pki_path>" {
-  capabilities = [ "read" ]
-}
+  path "/sys/mounts/<intermediate_pki_path>/tune" {
+    capabilities = [ "update" ]
+  }
+  ```
 
-path "/sys/mounts/<intermediate_pki_path>/tune" {
-  capabilities = [ "update" ]
-}
+  </CodeBlockConfig>
 
-# -----------------------------------------------------------------------------
-# Needed for Consul to use the externally-managed, mounted PKI secrets engines
+1. Add the following rules to you policy file so that Consul can use the  externally-managed PKI secrets engines that is already mounted: 
 
-path "/<root_pki_path>/" {
-  capabilities = [ "read" ]
-}
+  <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
-path "/<root_pki_path>/root/sign-intermediate" {
-  capabilities = [ "update" ]
-}
+  ```hcl
+  path "/<root_pki_path>/" {
+    capabilities = [ "read" ]
+  }
+  
+  path "/<root_pki_path>/root/sign-intermediate" {
+    capabilities = [ "update" ]
+  }
 
-path "/<root_pki_path>/*" {
-  capabilities = [ "create", "read", "update", "delete", "list" ]
-}
+  path "/<root_pki_path>/*" {
+    capabilities = [ "create", "read", "update", "delete", "list" ]
+  }
+  ```
+  </CodeBlockConfig>
 
-# Needed only when changing the CA provider from Vault with this RootPKIPath
-# to either:
-# 1. A different CA provider (such as Consul's built-in provider)
-# 2. Changing the RootPKIPath used by the Vault CA provider
-#
-# SECURITY NOTE: Because this is an extremely privileged endpoint,
-# We recommend only temporarily granting this permission if there's a need
-# to modify Consul's CA provider configuration as in (1) or (2).
-# That's why this permission is currently commented out.
-#
-# path "<root_pki_path>/root/sign-self-issued" {
-#  capabilities = [ "sudo", "update" ]
-#}
-
-# -----------------------------------------------------------------------------
-# Needed for Consul to renew its Vault token (if renewable) whether provided
-# directly in the CA provider config or using an auth method.
-
-path "auth/token/renew-self" {
-  capabilities = [ "update" ]
-}
+1. Only add the following rules to your policy file if you are changing the CA provider from Vault to a different CA provider (such as Consul's built-in provider) or changing the RootPKIPath used by the Vault CA provider:
 
-path "auth/token/lookup-self" {
-  capabilities = [ "read" ]
-}
-```
+  <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
-</CodeBlockConfig>
+  ```hcl
+  path "<root_pki_path>/root/sign-self-issued" {
+    capabilities = [ "sudo", "update" ]
+  }
+  ```
+  </CodeBlockConfig>
+  
+  Note that because this is an extremely privileged endpoint, we recommend only temporarily granting this permission if you need to modify Consul's CA provider configuration.
 
-### Consul Managed PKI Paths
```

Comment:
This headings are often linked to. We need to create a link alias to avoid breaking existing links.